src/test/nodestore/Backend_test.cpp

```diff
@@ -22,10 +22,12 @@
 #include <ripple/nodestore/Manager.h>
 #include <ripple/unity/rocksdb.h>
 #include <algorithm>
+#include <memory>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed